### ISSUE TITLE
Ubuntu update breaks PDF derivative generation

### ISSUE THREAD

**Issue by seanupton** [marriott-library/newspaper_works#69]
Re work implemented in #28, we now have a situation where an Ubuntu security update disabled PDF creation without editing policy.xml on platform:

https://usn.ubuntu.com/3785-1/

Related: https://usn.ubuntu.com/3768-1/

We need to modify PDF creation process, if possible, to route around this.  Ideally, it would be nice to create PDFs on default Ubuntu without changing system-wide configuration, and we need to determin if there is another command-line mechanism to do so (e.g. inkscape, ghostscript directly?).

**Comment by ebenenglish** [comment]
For anyone dealing with this in your development environment, there are several solutions posted on StackOverflow:

https://stackoverflow.com/questions/42928765/convertnot-authorized-aaaa-error-constitute-c-readimage-453

https://stackoverflow.com/questions/52703123/override-default-imagemagick-policy-xml

For the moment, I've edited `/etc/ImageMagick-6/policy.xml` like so:
```
<policy domain="coder" rights="read|write" pattern="LABEL" />
<policy domain="coder" rights="read|write" pattern="PDF" />
```

**Comment by ebenenglish** [comment]
TODO:

1. Add info to README
2. Add fix to travis.yml so builds don't break

**Comment by ebenenglish** [comment]
Fixed via #77.
